src/Compilers/CSharp/Portable/Symbols/AliasSymbol.cs

```diff
@@ -124,7 +124,7 @@ public override string Name
         {
             get
             {
-                return _aliasName.ValueText;
+                return _aliasName.ValueText ?? string.Empty;
```

Comment:
@jaredpar I'm guessing this is the expected behavior based on the recent change to `ValueText`

Reply:
Which recent changes are you referring to?

Reply:
`ValueText` was oblivious when I first made this PR. Now it returns a nullable string.